.gitignore

```diff
@@ -11,3 +11,5 @@ sdk/nodejs/node_modules
 *.iml
 
 yarn.lock
+.DS_Store
+README-BOILERPLATE.md
```

Comment:
That's what the instructional README gets renamed to toward the end of the process.  It's intended to be kept in case the author wants to refer to it.  I can remove the entry from `.gitignore` if you think it's weird or unnecessary.

Reply:
yeah I saw this at the bottom and forgot to remove this specific comment